CHANGELOG.md

```diff
@@ -10,9 +10,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 ### Added
+- Add in-enclave segfault backtrace log messages when the enclave is configured with Debug=1 and CapturePFGPExceptions=1, and the in-enclave logging is enabled with level=INFO.
```

Comment:
Typically, unhandled SEGV is catastrophic within an enclave, and the enclave cannot execute after that. It is thus better to log it always; not just when the log-level is INFO.

Our clients may not always use INFO logging level; but they still want to see important logs. See https://github.com/openenclave/openenclave/pull/4570